blazor/datagrid/row.md

```diff
@@ -151,4 +151,111 @@ Please refer to the following example.
 ```
 
 
-![Customizing Alternate Row in Blazor DataGrid](./images/blazor-datagrid-alter-row-customization.png)
\ No newline at end of file
+![Customizing Alternate Row in Blazor DataGrid](./images/blazor-datagrid-alter-row-customization.png)
+
+## Master Detail DataGrid
+
+The DataGrid has an option to visualize details of a record in another DataGrid in a master and detail manner. By simply clicking the master DataGrid row, the detail DataGrid will be populated with the corresponding data. This can be achieved as follows:
```

Comment:
The DataGrid has an option to visualize details of a record in another DataGrid in a master and detailed manner. By clicking the master DataGrid row, the detail DataGrid will be populated with the corresponding data. This can be achieved as follows: